contracts/socket/andromeda-socket-astroport/README.md

```diff
@@ -0,0 +1,365 @@
+# Andromeda Astroport Socket Contract
```

Comment:
I thought we agreed to keep detailed docs in the docs website instead of the README file